com.note_slayer.app.yaml

```diff
@@ -0,0 +1,43 @@
+app-id: com.note_slayer.app
```

Comment:
I will reiterate what you ignored:

I think the app-id should be `com.note_slayer.NoteSlayer`. `app` isn't an application name.